student-cvs/r4vc31-7548/index.html

```diff
@@ -0,0 +1,115 @@
+                                    [ESPOL]</a>
+                                (May
+                                2019 - Now)</li>
+                            <li>Status. – <span>In progress</span></li>
+                        </ul>
+                    </li>
+                </ul>
+            </article>
+
+            <article>
+                <h3><span>COURSES</span></h3>
+                <ul>
+                    <li><a
+                            href="https://app.aluracursos.com/user/r4vc31/fullCertificate/fda4615e9cc318308e663ca7825067cf">ONE
+                            Program - Alura Latam (aluracursos.com)</a></li>
+                    <li><a href="https://platzi.com/p/The_R-paradox/ruta/8860-pensamiento-logico-data/diploma/detalle/">Pensamiento
+                            lógico (platzi.com)</a></li>
+                    <li><a
+                            href="https://www.linkedin.com/learning/certificates/d73ee04227dcd712017acbf5a60e7cb51741a4d3e10bb5f1eb1016f020e9fffb?u=92964874">Conviértete
+                            en especialista en desarrollo Python (linkedin.com/learning)</a></li>
+                </ul>
+            </article>
+        </section>
+
+        <section class="contact">
+            <h2>Contact</h2>
+            <form action="#" method="post">
+                <div>
+                    <label for="name">Name: </label><input id="name" name="name" type="text" required>
+                </div>
+                <div>
+                    <label for="mail">E-mail: </label><input id="mail" name="mail" type="email" required>
+                </div>
+                <div>
+                    <label for="telephone">Phone Number: </label><input id="telephone" name="telephone" type="tel"
+                        required>
+                </div>
+
+                <div>
+                    <label for="project-type">Project Type</label><select name="project-type" id="project-type"
+                        required>
+                        <option value="" selected disabled>Select an option</option>
+                        <option value="webSite">WebSite</option>
+                        <option value="webApp">WebApp</option>
+                    </select>
+                </div>
+
+                <div>
+                    <fieldset>
+                        <legend>User Type</legend>
+                        <label for="particular">
+                            <input type="radio" name="user-type" id="particular" value="particular" required>Particular
+                        </label>
+                        <label for="company">
+                            <input type="radio" name="user-type" id="company" value="company">Company
+                        </label>
+                    </fieldset>
+                </div>
+                <div>
+                    <button type="submit">SEND</button>
+                </div>
+            </form>
+        </section>
+
+    </main>
+    <footer></footer>
```

Comment:
```suggestion
    <footer></footer>
```

eliminar este Footer vacio.

Reply:
Muchas gracias por las recomendaciones, subiré los cambios indicados.